sunpy/data/manager/downloader.py

```diff
@@ -0,0 +1,20 @@
+from abc import ABCMeta, abstractmethod
+
+
+class DownloaderBase(metaclass=ABCMeta):
+    @abstractmethod
+    def download(self, url, path):
+        """
+        Downloads and returns the path
```

Comment:
```suggestion
        Downloads and returns the path.
```

Also missing the rest of the docstring.